tests/system/action/motion/test_set_number_mixin.py

```diff
@@ -0,0 +1,1391 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class MotionSetNumberMixinTest(BaseActionTestCase):
```

Comment:
@jsangmeister I do not know the style in the backend: I would like to see more setup-functions and wrappers, e.g. for `self.client.post`. If it is always motion create this can be put inside an post-wrapper. The testcases itself are very large. Maybe - depending on the style - you can give Ludwig more hints about organizing the test code better.

Reply:
Yes, so far most tests were short enough that it didn't really matter. But for bigger tests (like this) this would be a good idea.